archai/common/common.py

```diff
@@ -36,7 +36,7 @@ def flush(self):
 def get_conf(conf:Optional[Config]=None)->Config:
     if conf is not None:
         return conf
-    return Config.get_global_instance()
+    return Config.get_inst()
```

Comment:
Can you review these changes, @gugarosa?

Reply:
Sure, looks good! This was a typo from a previous PR.

`get_inst()` is the correct method.

Reply:
thanks for confirming.